src/index.html

```diff
@@ -3,10 +3,380 @@
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <h3 class="card__reviews">Reviews: 5</h3>
+          </div>
+          <div class="card__price">
+            <p>Price:</p>
+            <p>$2199</p>
+          </div>
+          <a
+            class="card__buy-link"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div
+          class="card catalog__card"
+        >
+          <img
+            class="card__img"
+            src="../src/images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+          >
+          <h2 class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__container-stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <h3 class="card__reviews">Reviews: 5</h3>
+          </div>
+          <div class="card__price">
+            <p>Price:</p>
+            <p>$2199</p>
+          </div>
+          <a
+            class="card__buy-link"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div
+          class="card catalog__card"
+        >
+          <img
+            class="card__img"
+            src="../src/images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+          >
+          <h2 class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__container-stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <h3 class="card__reviews">Reviews: 5</h3>
+          </div>
+          <div class="card__price">
+            <p>Price:</p>
+            <p>$2199</p>
+          </div>
+          <a class="card__buy-link">
+            Buy
+          </a>
+        </div>
+
+      </section>
+    </main>
+
   </body>
+
+  <script>
+    let menuBtn = document.querySelector('.menu-btn');
+    let menu = document.querySelector('.menu');
+    menuBtn.addEventListener('click', function(){
+      menu.classList.toggle('active');
+    })
+
+    let page = document.querySelector('.page');
+    menuBtn.addEventListener('click', function(){
+      page.classList.toggle('page--open-menu');
+    })
```

Comment:
It's cool that I decided to add such a feature 🔥 
But then when you open the page, you should first have a burger icon, which will change to a cross after clicking and vice versa.

Reply:
Not fixed